src/test/integration/nft.extendedattributes.e2e-spec.ts

```diff
@@ -0,0 +1,115 @@
+import { CachingService } from 'src/common/caching/caching.service';
+import { EsdtModule } from 'src/endpoints/esdt/esdt.module';
+import { NftExtendedAttributesService } from 'src/endpoints/nfts/nft.extendedattributes.service';
+import { Test } from '@nestjs/testing';
+import { Constants } from 'src/utils/constants';
+import Initializer from './e2e-init';
+
+describe('Nft Extended Attributes Service', () => {
+  let nftExtendedAttributesService: NftExtendedAttributesService;
+
+  beforeAll(async () => {
+    await Initializer.initialize();
+
+    const moduleRef = await Test.createTestingModule({
+      imports: [EsdtModule],
+    }).compile();
+
+    nftExtendedAttributesService = moduleRef.get<NftExtendedAttributesService>(NftExtendedAttributesService);
+
+  }, Constants.oneHour() * 1000);
```

Comment:
Also here the initialization should be removed

Reply:
Fixed